### PR TITLE
pkp/pkp-lib#4042 record new rounds with own decision type

### PR DESCRIPTION
**Pull request by ajnyga**
https://github.com/pkp/pkp-lib/issues/4042

### CODE REVIEW

File: locale/en_US/submission.po [hunk 1]
```diff
@@ -1475,6 +1478,9 @@ msgstr "Revision required."
 msgid "notification.type.editorDecisionResubmit"
 msgstr "Resubmit for review."
 
+msgid "notification.type.editorDecisionNewRound"
+msgstr "New review round."
```
Comment (NateWr) [inline locale/en_US/submission.po]: > A new round of reviews was initiated.

Comment (ajnyga) [reply locale/en_US/submission.po]: fixed

Comment (ajnyga) [reply locale/en_US/submission.po]: Probably no need to restarts tests for this?

Comment (NateWr) [reply locale/en_US/submission.po]: Nope.

**Pull request merged**